### PR TITLE
Updated SOA 12.2.1.3 readme with latest info

### PR DESCRIPTION
**Pull request by sbattagi**
Hi,

We are in the process of uploading the latest SOA 12.2.1.3.0 image to OCR and updated the README.md with the latest information. This README.md will be pushed to the OCR for the user's reference to set up the SOA domain using docker.
Please review and approve this pull request.

Thanks  
Samba

### CODE REVIEW

File: OracleSOASuite/dockerfiles/12.2.1.3/README.md [hunk 1]
```diff
@@ -36,15 +36,15 @@ If that returns a username (which is the first field), you can skip the below us
...
+    # mkdir -p /u01/DockerVolume/SOAVolume/SOA
+    # chown -R 1000:1000 /u01/DockerVolume/SOAVolume/
+    # chmod -R 700 /u01/DockerVolume/SOAVolume/
 
 # Database
 
 You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
 
-Run the database container to host the RCU schemas using below steps:
+Run the database container to host the RCU schemas using below steps. For creating an Oracle Database container that uses data volumes to persist data, refer Oralce Enterprise Database documentation available at `https://container-registry.oracle.com`.
```
Comment (Djelibeybi) [inline OracleSOASuite/dockerfiles/12.2.1.3/README.md:47]: Typo in "Oracle"; also, you should refer to the documentation on GitHub as well, as some users may prefer to create their own image for Oracle Database using a more recent release than the one available on Container Registry.

File: OracleSOASuite/dockerfiles/12.2.1.3/README.md [hunk 2]
```diff
@@ -36,15 +36,15 @@ If that returns a username (which is the first field), you can skip the below us
 
 Once the `oracle` user is created, run the following commands as a root user:
 
-    # mkdir -p /u01/DockerVolume/SOAVolume/
-    # chown 1000:1000 /u01/DockerVolume/SOAVolume/
-    # chmod 700 /u01/DockerVolume/SOAVolume/
+    # mkdir -p /u01/DockerVolume/SOAVolume/SOA
+    # chown -R 1000:1000 /u01/DockerVolume/SOAVolume/
```
Comment (Djelibeybi) [inline OracleSOASuite/dockerfiles/12.2.1.3/README.md:40]: This doesn't take into account the scenario where the `getent` command returns a value that isn't `oracle` and the `gid` of the existing user isn't `1000`.  You probably don't need to set the `gid` here anyway, given the value of the `chmod` command next.

File: OracleSOASuite/dockerfiles/12.2.1.3/README.md [hunk 3]
```diff
@@ -36,15 +36,15 @@ If that returns a username (which is the first field), you can skip the below us
...
+    # chmod -R 700 /u01/DockerVolume/SOAVolume/
```
Comment (Djelibeybi) [inline OracleSOASuite/dockerfiles/12.2.1.3/README.md:41]: Setting the permissions to `700` removes access to the group so there's no need to include `:1000` above. Though, I would suggest that `750` makes a little more sense (that gives write permission to the owner with read permission to the group).

File: OracleSOASuite/dockerfiles/12.2.1.3/README.md [hunk 4]
```diff
@@ -58,8 +58,7 @@ The Oracle database server container requires custom configuration parameters fo
 
 	`DB_BUNDLE=basic`
 
-	`$ docker run -d --name soadb --network=SOANet -p 1521:1521 -p 5500:5500 -v /u01/DockerVolume/SOAVolume/DB:/opt/oracle/oradata --env-file ./db.env.txt -it --shm-size="8g" container-registry.oracle.com/database/enterprise:12.2.0.1`
-
+	`$ docker run -d --name soadb --network=SOANet -p 1521:1521 -p 5500:5500 --env-file ./db.env.txt -it --shm-size="8g" container-registry.oracle.com/database/enterprise:12.2.0.1`
```
Comment (Djelibeybi) [inline OracleSOASuite/dockerfiles/12.2.1.3/README.md:61]: You removed the volume from this command which means that if the Database container is removed, the data will disappear. This is probably not what you intended.

**Comment by Djelibeybi** [comment]
Also, please update the `CODEOWNERS` file and add an entry for `OracleSOASuite` to point to the right person to review future pull requests.

**Comment by sbattagi** [comment]
Hi @Djelibeybi 

Could you please review the latest changes?
Also please add @rdas0405  and @vivek-sam to this review.

Thanks
Samba

File: OracleSOASuite/dockerfiles/12.2.1.3/README.md [hunk 5]
```diff
@@ -36,15 +36,17 @@ If that returns a username (which is the first field), you can skip the below us
...
 
-Run the database container to host the RCU schemas using below steps:
+To create an Oracle Database container, you can either use the pre-built Oracle Database image available at `https://container-registry.oracle.com` or build a new Oracle Database image by following the [Oracle Database](https://github.com/oracle/docker-images/tree/master/OracleDatabase) github documentation.
+
+Below steps creates an Oracle Database container (using `container-registry.oracle.com/database/enterprise:12.2.0.1` image) without attaching data volumes to persist the data. In this case the data in the database will be lost when the container is stopped. For creating an Oracle Database container that uses data volumes to persist the data, refer the Oracle Enterprise Database documentation available at `https://container-registry.oracle.com` or [Oracle Database](https://github.com/oracle/docker-images/tree/master/OracleDatabase) github documentation.
```
Comment (Djelibeybi) [inline OracleSOASuite/dockerfiles/12.2.1.3/README.md:49]: Please change "Oracle Enterprise Database" to "Oracle Database 12c Enterprise Edition" and "github" to "GitHub".

File: OracleSOASuite/dockerfiles/12.2.1.3/README.md [hunk 6]
```diff
@@ -36,15 +36,17 @@ If that returns a username (which is the first field), you can skip the below us
...
 
 The Oracle database server container requires custom configuration parameters for starting up the container. These custom configuration parameters correspond to the datasource parameters in the SOA image to connect to the database running in the container.
```
Comment (Djelibeybi) [inline OracleSOASuite/dockerfiles/12.2.1.3/README.md:51]: Please change "Oracle database" to "Oracle Database"

**Review by rdas0405 — APPROVED** [review]

**Comment by sbattagi** [comment]
Hi @Djelibeybi 

This PR was already went thru the review process long back, but somehow merge was not happened.
Could you please help merging this PR ? 

Thanks
Samba

**Review by vivek-sam — COMMENTED** [review]
Changes look fine to merge.

**Comment by Djelibeybi** [comment]
It was merged 10 hours ago.

**Pull request merged**